nearcore/src/runtime/mod.rs

```diff
@@ -730,7 +727,7 @@ impl RuntimeAdapter for NightshadeRuntime {
         let mut total_size = 0u64;
         // TODO: Update gas limit for transactions
         let transactions_gas_limit = gas_limit / 2;
-        let mut transactions = vec![];
+        let mut result = PreparedTransactions { transactions: Vec::new(), limited_by: None };
```

Comment:
mini nit: You can consider implementing PreparedTransactions::new or ::default, potentially with just a macro.

Reply:
I like the current version a bit more.
Looking at it, it's immediately obvious that the first argument describes the transactions and the second describes a limit.
With `PreparedTransactions::new()` it wouldn't be clear what `None` means.
It's not that long and slightly more readable this way, I would keep it as is.